### PR TITLE
Made urls.py docstring consistent with other files in project template.

### PR DESCRIPTION
**Pull request by jarek-pacocha**
Corrected urls.py template docstring.

Adjusted a docstring in urls.py template by making the body 
start from the second line. This is consistent with other templates.

### CODE REVIEW

File: django/conf/project_template/project_name/urls.py-tpl [hunk 1]
```diff
@@ -1,4 +1,5 @@
-"""{{ project_name }} URL Configuration
+"""
+{{ project_name }} URL Configuration
```
[the remark this answers is not in the document]
Comment (jarek-pacocha) [reply django/conf/project_template/project_name/urls.py-tpl:2]: This makes sense. 

Tell me, should I commit this suggestion and add it to the batch or do it differently? Thanks

[the remark this answers is not in the document]
Comment (felixxm) [reply django/conf/project_template/project_name/urls.py-tpl:2]: You can push the 2nd commit to your branch.

**Comment by felixxm** [comment]
@jarek-pacocha Thanks :+1: Welcome aboard :boat:

**Comment by jarek-pacocha** [comment]
Good to be here 🙂👍

**Pull request merged**